challenges.tex

```diff
@@ -0,0 +1,540 @@
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l||l|l||l|l}
+  & \multicolumn{4}{c||}{Ring Costs}
+  & \multicolumn{2}{c}{\multirow{2}{*}{Color Assignment }}\\\cline{2-5}
+  & Color  & Price & Color  & Price & \multicolumn{2}{c}{}\\\cline{2-7}
+  & Yellow & 0 & Green & 0
+  & RS1: $\colconfig_1$ & RS1: $\colconfig_2$ \\
+  & Blue  & 1 & Orange & 2
+  & RS2: $\colconfig_2$ & RS2: $\colconfig_1$ \\\hline\hline
+  Configuration & \multicolumn{2}{c||}{$\colconfig_1$}
+  & \multicolumn{2}{c||}{$\colconfig_2$}
+  & $\texttt{option1}$ & $\texttt{option2}$\\
+ \end{tabular}
+ \caption{Materials}
+ \label{tab:ring-costs}
+\end{table}
+
+\paragraph{Materials}\label{sec:materials}
+The available material that can be used per challenge is restricted
+(unless stated otherwise) per machine according to the information in
+\reftab{tab:materials}.
+\begin{table}[!htb]
+ \centering
+  \begin{tabularx}{\linewidth}{l|l}
+   Machine & Available Material  \\\hline
+   \ac{BS} & 2 bases of each color \\
+   \ac{CS} & 3 cap-carriers (cap color choices up to each team)  \\
+   \ac{RS} & 4 rings of each assigned color (8 in total)  \\
+  \end{tabularx}
+ \caption{Materials}
+ \label{tab:materials}
+\end{table}
+
+\paragraph{Orders}
+Unless specified otherwise, orders that have to be fulfilled in challenges
+are entered through the web shop
+(see the RefBox workshop in \refsec{sec:qualification})
+by any member of the competing team.
+
+In challenges where only one \ac{RS} is present, teams are responsible to
+order products which can be assembled using the available stations only.
+
+\paragraph{Scoring}\label{sec:scoring}
+While the RefBox may assign points during challenges according to the
+regular RCLL rules, those points do not count towards this competition.
+Instead, the point scoring for each challenge is listed in
+\refsec{sec:challenges}.
+
+\subsection{Field Operators}\label{sec:operators}
+In challenges where mockup machines are used, the actual assembly steps have
+to be performed by human supervisors. Whenever a machine is instructed,
+the RefBox operator announces the required interaction. One field operator may
+proceed to enter the field in order to perform the interaction. Afterwards the
+field has to be left immediately.
+The usual rules for replenishing resources (respecting the limited materials
+\refsec{sec:materials}) apply.
+
+\subsection{Available Challenges for the Primary Competition}
+\label{sec:challenges}
+Challenges have different types and variations (difficulty levels).
+The overall score of the competition is calculated by summing up the score
+in the highest difficulty achieved in each of the challenge types.
+The challenge types of the competition are described in
+\refsec{sec:challenge-navigation}-\ref{sec:challenge-markerless}.
+
+The RefBox is used to log the progress and data for each challenge.
+Participating teams are required to send the corresponding game-report dumps
+to the \ac{OC}.
```

Comment:
It would be helpful to have more information (in the rulebook addendum) on how to create the game report.

nit:
```suggestion
Participating teams are required to send the corresponding game report dumps
to the \ac{OC}.
```

Reply:
This is covered by the refbox tutorial video